app/views/hyrax/dashboard/collections/_show_add_items_actions.html.erb

```diff
@@ -1,16 +1,21 @@
 <h2 class="sr-only"><%= t('hyrax.collection.actions.header') %></h2>
 <div class="actions-controls-collections">
   <% if can? :edit, presenter.solr_document # TODO: Shouldn't this be :deposit -- dependency on participants %>
-    <%= link_to t('hyrax.collection.actions.add_works.label'),
+    <%= link_to t('hyrax.collection.actions.add_existing_works.label'),
                 hyrax.my_works_path(add_works_to_collection: presenter.id),
-                title: t('hyrax.collection.actions.add_works.desc'),
+                title: t('hyrax.collection.actions.add_existing_works.desc'),
                 class: 'btn btn-default pull-right',
                 data: { turbolinks: false } %>
+    <%= link_to t('hyrax.collection.actions.add_new_work.label'),
+                '#',
+                title: t('hyrax.collection.actions.add_new_work.desc'),
+                data: { behavior: "select-work", target: "#worktypes-to-create", 'create-type' => 'single', add_works_to_collection: presenter.id },
```

Comment:
And this shouldn't require the `data-turbolinks` jazz, right?

Reply:
Here it is correct to not include `data-turbolinks`.  The work type selection modal will not display if you include it.